src/PointwiseFunctions/AnalyticSolutions/Errors.hpp

```diff
@@ -0,0 +1,73 @@
+// Distributed under the MIT License.
```

Comment:
The compute tag is specific to evolution systems. Please move it to `evolution`, I suggest `Evolution/ComputeTags.hpp`. Then you can move the `Errors` prefix tag to `AnalyticSolutions/Tags.hpp`.

Reply:
@nilsdeppe since you asked me to put this in AnalyticSolutions/Errors.hpp, are you ok with @nilsleiffischer‘s suggestion that I put the compute tag and test back in Evolution/ComputeTags, and move it back into the evolution namespace?